build/constants.yaml

```diff
@@ -137,12 +137,12 @@
     php81-version: 8.1.0
     php81-keys: 528995BFEDFBA7191D46839EF9BA0ADA31CBD89E 39B641343D8C104B2B146DC3F9C39DC0B9698544
     php81-tar-sha256: a1317eff0723a2b3d3122bbfe107a1158570ea2822dc35a5fb360086db0f6bbc
-    php80-version: 8.0.13
```

Comment:
Hi @qianz2 do we need to updated the php tag https://github.com/microsoft/Oryx/blob/8ff689840b634cc2bfdd90942d5a1ab808e64b41/build/constants.yaml#L126 for the new runtimes?

Reply:
Updated, thanks